tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
+    "should successfully parse" >> {
 
-  "parseRelFile - ./image.png" in {
-    parseRelFile("./image.png") must beSome(file("image.png"))
-  }
+      """simple file name with extension such as "image.png"""" in {
```

Comment:
I'm not really sold on this concept.

For one thing, why use triple-double-quotes, when you could just pick some other way to set off the file name, say backticks?

More importantly, the phrase "such as" suggests that this is testing more than it really is. In fact it's testing parsing of exactly that one path, no more and no less. I would prefer either "simple file name with extension" (which describes the value without showing it) or even just "image.png", which would read fine I think.